docs/third-party-developers/extensibility/data-store/collections.md

```diff
@@ -0,0 +1,84 @@
+# Collections Store <!-- omit in toc -->
+
+## Table of Contents <!-- omit in toc -->
+
+-   [Overview](#overview)
+-   [Usage](#usage)
+-   [Actions](#actions)
+    -   [receiveCollection( namespace, resourceName, queryString, ids = \[\], items = \[\], replace = false )](#receivecollection-namespace-resourcename-querystring-ids---items---replace--false-)
+-   [Selectors](#selectors)
+    -   [getCollection](#getcollection)
+    -   [getCollectionHeader](#getcollectionheader)
+
+## Overview
+
+...
```

Comment:
Can we please add an overview with real-life use cases as to why developers would want to interact with it?